### PR TITLE
ci: static fixtures

### PR DESCRIPTION
**Pull request by matthewhudson**
- [x] Have you followed the [contributing guidelines](https://github.com/microservices/.github/blob/master/CONTRIBUTING.md)?
- [ ] Have you explained what your changes do?

**Please note: we will close your PR without comment if you do not check the
boxes above and provide ALL requested information.**

---

### CODE REVIEW

File: tests/regression/test.sh [hunk 1]
```diff
@@ -23,7 +23,7 @@ for repo in clojure d go node java python php ruby rust scala ; do
   cd ..
 done
 
-for repo in python-events node-events ; do
+for repo in node-events ; do
```
Comment (matthewhudson) [inline tests/regression/test.sh]: Temporary test. I can get `node-events` and `python-events` working locally, but they don't seem to be passing tests in CI.

Comment (StoryscriptHuman) [reply tests/regression/test.sh]: Linux networking fix for subscriptions coming this monday

Comment (matthewhudson) [reply tests/regression/test.sh]: @StoryscriptHuman is working on a patch

Comment (StoryscriptHuman) [reply tests/regression/test.sh]: Available in https://github.com/microservices/oms/pull/347

**Pull request merged**